ros2_batch_job/packaging.py

```diff
@@ -112,9 +112,21 @@ def build_and_test_and_package(args, job):
     # Only on Linux and OSX Python scripts have a shebang line
     if args.os in ['linux', 'osx']:
         print('# BEGIN SUBSECTION: rewrite shebang lines')
+        filenames = []
```

Comment:
Nitpick: it might be good to name this variable `paths` since it contains full paths rather than just filenames.